ci-operator/step-registry/install-operators/install-operators-commands.sh

```diff
@@ -26,22 +33,20 @@ for operator_obj in "${OPERATOR_ARRAY[@]}"; do
     operator_group=$(jq --raw-output '.operator_group // ""' <<< "$operator_obj")
     operator_target_namespaces=$(jq --raw-output '.target_namespaces // ""' <<< "$operator_obj")
 
-    # If install_namespace not defined, exit.
-    if [[ -z "${operator_install_namespace}" ]]; then
-        echo "ERROR: install_namespace is not defined"
-        exit 1
-    fi
-
     # If name not defined, exit.
     if [[ -z "${operator_name}" ]]; then
         echo "ERROR: name is not defined"
         exit 1
     fi
 
-    # If channel is not defined, exit.
+    # If install_namespace not defined, use DEFAULT_OPERATOR_INSTALL_NAMESPACE.
+    if [[ -z "${operator_install_namespace}" ]]; then
+        operator_install_namespace="${DEFAULT_OPERATOR_INSTALL_NAMESPACE}"
+    fi
+
+    # If channel is not defined, use DEFAULT_OPERATOR_CHANNEL.
```

Comment:
Do you want to handle the situation when channel is not defined, but some other channel(eg stable-x.y like [here](https://github.com/Simranpal/release/blob/5fad04cec9ef18fa08b65993d331fe4118929457/ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.15-lp-interop.yaml#L48)) is being used other than default? or will using default set that channel

Reply:
I want to provide the user the option to not specify the channel, so when it's not mentioned, it'll automatically set with `!default`. Than our script is picking the default channel from the marketplace (might be stable-x.y or anything else).

If channel is mentioned, like in your ref- sets to `"channel": "stable-2.4"`, it'll not use the default value "!default". Hope that answered your question :)

Reply:
okay yes, I wanted to know if it will pick the default channel (like example above) from marketplace.
Thanks!